spec/system/publishers/publishers_can_publish_a_vacancy_as_a_school_spec.rb

```diff
@@ -149,118 +161,7 @@
       expect(current_path).to eq(organisation_job_summary_path(created_vacancy.id))
     end
 
-    scenario "saving and finishing later" do
```

Comment:
I deleted this as I don't think this test provides much/anything that the previous test does not